src/services/minioService.ts

```diff
@@ -0,0 +1,99 @@
+import { Client } from 'minio';
+
+interface MinioConfig {
+  endPoint: string;
+  port: number;
+  useSSL: boolean;
+  accessKey: string;
+  secretKey: string;
+  bucket: string;
+}
+
+const MINIO_CONFIG: MinioConfig = {
+  endPoint: process.env.MINIO_ENDPOINT || 'localhost',
+  port: parseInt(process.env.MINIO_PORT || '9000', 10),
+  useSSL: process.env.MINIO_USE_SSL === 'true',
+  accessKey: process.env.MINIO_ACCESS_KEY || 'minioadmin',
+  secretKey: process.env.MINIO_SECRET_KEY || 'minioadmin',
+  bucket: process.env.MINIO_BUCKET || 'excalidraw',
+};
+
+export class MinioService {
```

Comment:
I don't see this being used for non-excalidraw files, there should be an env var to control which storage backend is used: regular files or minio.